content/en/docs/components/pipelines/installation/compatibility-matrix.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Compatibility Matrix"
+description = "Kubeflow Pipelines compatibility matrix with related software"
+weight = 50
++++
+
+## Kubeflow Pipelines Backend and TFX compatibility
+
+| Kubeflow Pipelines Backend & [TFX](https://www.tensorflow.org/tfx) Version | <=0.30.0 | 1.0.0 | 1.2.0 |
```

Comment:
Is it possible to add HTML to DOCSY?


<table>
<thead>
<tr>
<th>Kubeflow Pipelines Backend</th>
<th colspan="3">[TFX](https://www.tensorflow.org/tfx) Version</th>
</tr>
</thead>
<tbody>
<tr>
<td> </td>
<td> <=0.30.0 </td>
<td> 1.0.0 </td>
<td> 1.2.0 </td>
</tr>
  
<tr>
<td> <=1.6 </td>
<td> Compatible </td>
<td> Not fully compatible[1] </td>
<td> Not fully compatible[1]️ </td>
</tr>
  
<tr>
<td> >=1.7 </td>
<td> Not fully compatible[1] </td>
<td> Not fully compatible[1]️ </td>
<td> Compatible </td>
</tr>
  
</tbody>
</table>